### PR TITLE
Robfallows/issue4

### LINKED ISSUE #4: Cheat Sheet is not a well formed file
### Problem Statement

The Markdown cheat sheet file is a badly formed file and needs to be restructured to prevent errors/warnings when the linter looks at it.  Also it does not allow the reader to easily see the 'tags' we are using/explaining.

### PR DESCRIPTION
**Pull request by robfallows**
## Problem

Closes #4.

## Description

This PR provides a curated GitHub flavoured markdown "cheatsheet".

## Pre-Requisites

None.

## Commitment to Test

- Testing takes 10 minutes.
- Reviewing 1 file.

## Test Instructions / Acceptance Criteria

- View the markdown document in its final, rendered form.
  - [x] There are no (obvious) typos/misspellings.
  - [x] It makes sense.
  - [x] Examples are rendered correctly.
  - [x] The Optiq image is rendered from a URL and an image file successfully.
- View the original (unrendered) file.
  - The only linting errors are:
    - [x] One MD025 (duplicate H1)
    - [x] One MD034 (bare URL)

### CODE REVIEW

File: markdown-cheat-sheet.md [hunk 1]
```diff
@@ -1,121 +1,252 @@
...
+## Text Formatting
 
-Here is some *italicized text*
+**Bold Text** is produced using `**Bold Text**`
 
-### Blockquote
+*Italic Text* is produced using `*Italic Text*`
 
-> blockquote
+You may of course have ***bold and italic*** text.
```
Comment (SimonEdmunds) [inline markdown-cheat-sheet.md:96]: Perhaps add the 'is produced using ' example as with the previous two

File: markdown-cheat-sheet.md [hunk 2]
```diff
@@ -1,121 +1,252 @@
...
 
-`code`
+This form is preferred: [Corporate Website](https://www.softwareone.com/)
 
-### Horizontal Rule
+---
+
+## Escape Sequence
+
+You can escape characters which would otherwise be actioned using \\ eg. \\\`
```
Comment (SimonEdmunds) [inline markdown-cheat-sheet.md:116]: Perhaps - 'You can escape character which would otherwise be actioned using \\ (a backslash)'

**Comment by SimonEdmunds** [comment]
Just a couple of comments.  If you want I can approve the pull request

File: markdown-cheat-sheet.md [hunk 3]
```diff
@@ -1,121 +1,252 @@
...
+This cheatsheet for Centiq is a simplified version of [this document](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax).
 
-## Basic Syntax
+Note that Markdown does have rules. Mostly, what you type will be rendered as you would expect. However when things end up looking unexpected, it's probably because you "broke a rule". I've attempted to explain the rules in this cheatsheet, but to aid their understanding you will have installed a "Markdown linting" extension which flags rule-breaking as you type.
 
-These are the elements outlined in John Gruber’s original design document. All Markdown applications support these elements.
+## Text Writing
 
-### Heading
+It is easy to write in GFMD. Just type simple text and use the below heading styles to mark the text and you are good to go!
```
[the remark this answers is not in the document]
Comment (SimonEdmunds) [reply markdown-cheat-sheet.md:11]: Also this comment is probably not in the correct place.  I think it got confused. I shall see if I can move it.

File: markdown-cheat-sheet.md [hunk 4]
```diff
@@ -1,121 +1,270 @@
...
+
+This form is preferred: [Corporate Website](https://www.softwareone.com/)
+
+---
 
-![alt text](https://www.markdownguide.org/assets/images/tux.png)
+## Escape Sequence
 
-## Extended Syntax
+You can escape characters which would otherwise be actioned by preceding each actionable character with a backslash (`\`). For example \\\`
```
Comment (SimonEdmunds) [inline markdown-cheat-sheet.md:116]: Not quite sure what the example is trying to show - is it meant to show an escaped ` character?

Comment (SimonEdmunds) [reply markdown-cheat-sheet.md:116]: Maybe the example could be \`An Example\` - escaping the back tick characters?

Comment (robfallows) [reply markdown-cheat-sheet.md:116]: It also renders properly in GitHub. The escape sequence you're referring to renders as a backslash followed by a backtick.

**Review by SimonEdmunds — APPROVED** [review]
All good

**Pull request merged**